### DISCUSSION TITLE
VTherm bascule en mode "Boost" après une detection d'ouverture

### DISCUSSION THREAD

**Discussion by kirtapix** [jmcollin78/versatile_thermostat#580] [Q&A]
Bonjour,
J'ai un comportement du Vtherm que je ne m'explique pas : lorsque une de mes fenêtres est ouverte, le Vtherm bascule sur le preset "Hors-Gel" (ce qui est configuré), cependant à la fermeture de la fenêtre, c'est le preset "Boost" qui est configurer. Est ce normal ? Cela ne devrait pas être le preset précédemment configurer avant que la fenêtre n'est été ouverte ?
Merci d'avance pour votre réponse.

**Comment by jmcollin78** [comment]
Hello @kirtapix ,

> Cela ne devrait pas être le preset précédemment configurer avant que la fenêtre n'est été ouverte ?

Si, c'est bien ce que ça fait : ça remet le preset précédent qui était actif au moment de la détection de la fenêtre ouverte.
Donne moi les attributs de ton VTherm lorsque la fenêtre est ouverte (Outils de dev / Etats).
Tu dois voir quelque-chose comme ça:
```
window_state: "on"
...
saved_preset_mode: eco
saved_target_temp: 15
```
